### PR TITLE
basic schema with working resolvers

### PR DESCRIPTION
**Pull request by EmperorEarth**

### CODE REVIEW

File: dev/001.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+use `newssite`;
+
+CREATE TABLE `post` (
```
Comment (robvdl) [inline dev/001.sql]: This looks like MySQL, John, I thought you (we) only use Postgres?

Comment (robvdl) [inline dev/001.sql]: Actually I have noticed that existing Go database migration tools are all low-level SQL based, which is fine if you are using only one DB system but not so much if you want to support multiple DB systems, where you have to provide different migration sets for each DB system.

Django for example does all it's migrations in Python to get around this, but I don't think such a library exists in Go.

Comment (johncurley) [inline dev/001.sql]: Yeah it's mysql as Kevin is used to MySql and we're just trying to get something going quickly.

Comment (robvdl) [inline dev/001.sql]: Fair enough then, I was just confused as we had migrated away from MySQL years ago already and never looked back really.

Also at work, we use Postgres for every new project, MySQL only for legacy apps.  Even for those legacy apps, MariaDB is preferred over MySQL these days.

But if it's just to get something running, fine I guess, it's mostly your project after all :)

**Comment by johncurley** [comment]
It's mainly so we have a base to work off.

**Pull request merged**